src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4442,17 +4685,19 @@ InvocationOperation makeInvocationDroppingInstanceForStaticMethods(IMethodSymbol
             InvocationOperation makeInvocation(SyntaxNode syntax, IMethodSymbol method, IOperation? instanceOpt, ImmutableArray<IArgumentOperation> arguments)
             {
                 Debug.Assert(method.IsStatic == (instanceOpt == null));
+                var visitedArguments = makeArguments(arguments, ref instanceOpt);
                 return new InvocationOperation(method, instanceOpt,
-                                                isVirtual: method.IsVirtual || method.IsAbstract || method.IsOverride,
-                                                makeArguments(arguments), semanticModel: null, syntax,
-                                                method.ReturnType, isImplicit: true);
+                                               isVirtual: method.IsVirtual || method.IsAbstract || method.IsOverride,
+                                               visitedArguments, semanticModel: null, syntax,
+                                               method.ReturnType, isImplicit: true);
             }
 
-            ImmutableArray<IArgumentOperation> makeArguments(ImmutableArray<IArgumentOperation> arguments)
+            ImmutableArray<IArgumentOperation> makeArguments(ImmutableArray<IArgumentOperation> arguments, ref IOperation? instanceOpt)
             {
                 if (arguments != null)
                 {
-                    return VisitArguments(arguments);
+                    (instanceOpt, var visitedArguments) = VisitArgumentsWithVisitedInstance(arguments, instanceOpt);
```

Comment:
>arguments [](http://example.com/codeflow?start=92&length=9)

Can this ever contain an IInterpolatedStringHandlerCreationOperation?

Reply:
I don't think so, but it seemed more robust to always go through this codepath.

Reply:
I opted to assume there will never be an interpolated string handler in this code.